smokey/src/main/java/frc/robot/common/SwerveDriveRotationMode.java

```diff
@@ -0,0 +1,16 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: public enum SwerveDriveRotationMode.java
+// Intent: Forms enum to specify the Drive Rotation Mode.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.common;
+
+public enum SwerveDriveRotationMode{
```

Comment:
Seems quite close to SwerveDriveMode ... which also has a 'Normal' flavor ...

Reply:
Will address by renaming this.  see comment above.

Reply:
changed enum name. let me know if the intent is now clear.